### PR TITLE
[Container Section 2 of 2] DCOS-12398: Consolidate container sections

### PR DESCRIPTION
**Pull request by mlunoe**
This PR consolidates the two container sections to use one.

~~Container Section 1 of 2: https://github.com/dcos/dcos-ui/pull/1770~~ Merged
Container Section 2 of 2: https://github.com/dcos/dcos-ui/pull/1771

~~Diff for review can be found here: https://github.com/dcos/dcos-ui/compare/mlunoe/bug-fix/DCOS-12398-align-container-section-with-doc-specification...mlunoe/bug-fix/DCOS-12398-align-container-section-with-doc-specification-2?expand=1~~ No longer relevant.

### CODE REVIEW

**Comment by mlunoe** [comment]
Rebased with integration test fixes

**Comment by mlunoe** [comment]
Restest this please

**Comment by mlunoe** [comment]
Restest this please

**Comment by mlunoe** [comment]
Run unit tests

**Comment by mlunoe** [comment]
Run integration tests

**Comment by mlunoe** [comment]
Run integration tests

**Comment by mlunoe** [comment]
Run integration tests

**Comment by mlunoe** [comment]
Run unit tests

**Comment by mlunoe** [comment]
Run unit tests

**Comment by orlandohohmeier** [comment]
🙇  Thanks a loft for consolidating the single and multi container form. This is a first step towards aligning the internal (from reducer) data structure, wich will give us more flexibility when working with form data.

**Comment by orlandohohmeier** [comment]
form-rendering-issue mov

There seems to be an rendering issue after switching from the multi to single container form.

**Comment by mlunoe** [comment]
All this PR does is consolidate the two sections.
The former PR takes care of the doc links and help text: https://github.com/dcos/dcos-ui/pull/1770

If you take a look at the diff I linked in the description you’ll see that is the only difference:
https://github.com/dcos/dcos-ui/compare/mlunoe/bug-fix/DCOS-12398-align-container-section-with-doc-specification...mlunoe/bug-fix/DCOS-12398-align-container-section-with-doc-specification-2?expand=1

The title was a little misleading, so updated that.

**Comment by mlunoe** [comment]
That error you pointed out was not introduced as a part of this or dependent PR.
I'll take a look at it in this PR chain as this touches the point where the issue was introduced anyway:
https://github.com/dcos/dcos-ui/pull/1853

File: plugins/services/src/js/components/modals/NewCreateServiceModal.js [hunk 1]
```diff
@@ -179,6 +179,7 @@ class NewServiceFormModal extends Component {
 
       case 'app':
         this.setState({
+          activeTab: null,
```
Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: This fixes the bug that @orlandohohmeier found

Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModal.js]: Removed from this PR and put into separate PR: #1857

File: plugins/services/src/js/components/forms/ContainerServiceFormSection.js [hunk 2]
```diff
@@ -399,6 +510,7 @@ ContainerServiceFormSection.defaultProps = {
 ContainerServiceFormSection.propTypes = {
   data: React.PropTypes.object,
   errors: React.PropTypes.object,
+  path: React.PropTypes.string,
```
Comment (MatApple) [inline plugins/services/src/js/components/forms/ContainerServiceFormSection.js]: #nit abc

Comment (mlunoe) [inline plugins/services/src/js/components/forms/ContainerServiceFormSection.js]: Done!

**Comment by MatApple** [comment]
Looks good, works good :) 👍

**Comment by orlandohohmeier** [comment]
LGTM! 🚢
